writers/streamingbatchwriter/streamingbatchwriter.go

```diff
@@ -285,7 +293,32 @@ func (w *StreamingBatchWriter) startWorker(ctx context.Context, errCh chan<- err
 		go wr.run(ctx, &w.workersWaitGroup, tableName)
 		ch <- m
 		return nil
+	case *message.WriteDeleteRecord:
+		w.workersLock.Lock()
+		defer w.workersLock.Unlock()
+		if w.deleteRecordWorker != nil {
+			w.deleteRecordWorker.ch <- m
+			return nil
+		}
+		ch := make(chan *message.WriteDeleteRecord)
+		flush := make(chan chan bool)
+		// TODO: flush all workers for nested tables as well
```

Comment:
hmm good point here; I think we may want to link this to a known issue, but can defer implementation because this writer is only used by file-type destinations right now (that won't support delete)